packages/cli/src/evaluation/test-runner/__tests__/test-runner.service.ee.test.ts

```diff
@@ -0,0 +1,120 @@
+import type { SelectQueryBuilder } from '@n8n/typeorm';
+import { stringify } from 'flatted';
+import { readFileSync } from 'fs';
+import { mock, mockDeep } from 'jest-mock-extended';
+import { mockInstance } from 'n8n-core/test/utils';
+import path from 'path';
+
+import { ActiveExecutions } from '@/active-executions';
+import type { ExecutionEntity } from '@/databases/entities/execution-entity';
+import type { TestDefinition } from '@/databases/entities/test-definition.ee';
+import type { User } from '@/databases/entities/user';
+import type { ExecutionRepository } from '@/databases/repositories/execution.repository';
+import type { WorkflowRepository } from '@/databases/repositories/workflow.repository';
+import { NotFoundError } from '@/errors/response-errors/not-found.error';
+import type { TestDefinitionService } from '@/evaluation/test-definition.service.ee';
+import type { WorkflowRunner } from '@/workflow-runner';
+
+import { TestRunnerService } from '../test-runner.service.ee';
+
+const wfUnderTestJson = JSON.parse(
+	readFileSync(path.join(__dirname, './mock-data/workflow.under-test.json'), { encoding: 'utf-8' }),
```

Comment:
Does this work? AFAIK we [mock FS access](https://github.com/n8n-io/n8n/blob/master/packages/cli/test/setup-mocks.ts#L8) in the tests at global level

Reply:
It seems we mock `node:fs`, but not the `fs`, this is why it works 🤔